### PR TITLE
feat: implement new currency-selector design

### PR DESCRIPTION
**Pull request by goldenmaya**
## Description
https://virtocommerce.atlassian.net/browse/ST-5043
### Artifact URL:
https://vc3prerelease.blob.core.windows.net/packages/vc-theme-b2b-vue-1.42.0-pr-819-d58f-d58fa936.zip

### CODE REVIEW

File: client-app/ui-kit/components/atoms/popover/vc-popover.vue [hunk 1]
```diff
@@ -1,147 +1,114 @@
...
+      ref="triggerNode"
+      :aria-describedby="`popover-${$.uid}`"
+      tabindex="0"
+      role="button"
+      @mouseenter="trigger === 'hover' && open()"
+      @mouseleave="trigger === 'hover' && close()"
+      @focus="trigger === 'hover' && open()"
+      @blur="trigger === 'hover' && close()"
+      @click="trigger === 'click' && toggle(!isShown)"
+      @keyup="trigger === 'click' && toggle(!isShown)"
```
Comment (asvishnyakov) [inline client-app/ui-kit/components/atoms/popover/vc-popover.vue:13]: Why not just call open/close too?

**Review by asvishnyakov — CHANGES_REQUESTED** [review]

File: client-app/shared/catalog/components/product-price-block.vue [hunk 2]
```diff
@@ -45,11 +45,7 @@
```
[the remark this answers is not in the document]
Comment (goldenmaya) [reply client-app/shared/catalog/components/product-price-block.vue:45]: it uses vc-popover

File: client-app/ui-kit/components/atoms/popover/vc-popover.vue (hunk 1 above)
Comment (goldenmaya) [reply client-app/ui-kit/components/atoms/popover/vc-popover.vue:13]: it has to toggle menu on click or keyup

File: client-app/shared/catalog/components/product-price-block.vue (hunk 2 above)
[the remark this answers is not in the document]
Comment (asvishnyakov) [reply client-app/shared/catalog/components/product-price-block.vue:45]: So VcPopover was refactored and you just moved title & closed buttom out of component

**Review by asvishnyakov — APPROVED** [review]

**Pull request merged**